### PR TITLE
Descend the test from `ContextAwareTest`

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR improves the test to employ `ContextAwareTest` API so that we don't have to create and release the context fixture manually.

### CODE REVIEW

File: src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java [hunk 1]
```diff
@@ -11,13 +12,11 @@
 import static io.spine.testing.TestValues.randomString;
 
 @DisplayName("Hello Context should")
```
Comment (yuri-sergiichuk) [inline src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java:14]: Shouldn't we backtick the name?

File: src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java [hunk 2]
```diff
@@ -11,13 +12,11 @@
...
-class HelloContextTest {
+class HelloContextTest extends ContextAwareTest {
```
Comment (yuri-sergiichuk) [inline src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java:15]: Can we make the class `final`?

Comment (alexander-yevsyukov) [reply src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java:15]: We probably can, but this is not customary in JUnit.

File: src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java (hunk 1 above)
Comment (alexander-yevsyukov) [reply src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java:14]: We don't use ticks here because none of the words are code.

**Pull request merged**